model.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+  attr_reader :definition, :flashcards
+  attr_accessor :term
+
+  def initialize(word = {})
+    @definition = word[:definition]
+    @term = word[:term]
+  end
+
+  def correct?(guess)
+    guess == self.term ? true : false
+  end
```

Comment:
Great method on your card, but the ternary is redundant.

This does the exact same:

```ruby
guess == term
```